x-pack/platform/plugins/shared/actions/server/routes/connector/error_handler.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { IKibanaResponse, KibanaResponseFactory } from '@kbn/core/server';
+import {
+  isResponseError as isElasticsearchResponseError,
+  isUnauthorizedError,
+  isNotFoundError,
+} from '@kbn/es-errors';
+import type { KibanaServerError } from '@kbn/kibana-utils-plugin/common';
+import { identifyEsError } from '@kbn/task-manager-plugin/server/lib/identify_es_error';
+
+function isKibanaServerError(error: unknown): error is KibanaServerError {
```

Comment:
Seems like this should live in the same module as `KibanaServerError` - I would expect other folks could make use of this ...